.env.example

```diff
@@ -0,0 +1,44 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:BRIrrEIO+jBwBksEztbwyPoE6r7/wCQjTisadad9oXM=
+APP_DEBUG=true
+APP_URL=http://localhost
+
+LOG_CHANNEL=stack
+
+DB_CONNECTION=mysql
+DB_HOST=road-database
```

Comment:
Host should be as a name of service (`database` from docker-compose), not a container. Please check whether it works before commit.

Reply:
It works even with container name, I've just forgotten to add commands to makefile to run migrations, but thank you now I know that I should use service name.

Fixed.

Reply:
Hm... Didn't know about it. Thanks!